### PR TITLE
Buildkite source + updates all Faros sources

### LINKED ISSUE #118: Buildkite Source

### PR DESCRIPTION
**Pull request by thomas-gerber**
# Description

Adds the [buildkite source](https://github.com/faros-ai/airbyte-connectors/tree/main/sources/buildkite-source).

Also updates all current Faros sources and harmonizes the documentUrl to the corresponding GitHub source.

Fixes #118 

## Type of change
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

# Checklist
- [x] Have you checked to there aren't other open Pull Requests for the same update/change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Comment by thomas-gerber** [comment]
Note: we are still looking into whether the `cicd_BuildCommitAssociation` skips should be dealt with in this PR or separately (Cc @cjwooo)

**Comment by thomas-gerber** [comment]
Documentation: https://community.faros.ai/docs/buildkite-source

**Comment by thomas-gerber** [comment]
> Note: we are still looking into whether the `cicd_BuildCommitAssociation` skips should be dealt with in this PR or separately (Cc @cjwooo)

Will be handled separately; see https://github.com/faros-ai/airbyte-connectors/issues/408

File: init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml [hunk 1]
```diff
@@ -141,8 +141,8 @@
 - sourceDefinitionId: "05f9f7d4-d900-4ebd-89fc-abcce64e342b"
   name: "Agile Accelerator"
   dockerRepository: "farosai/airbyte-agileaccelerator-source"
-  dockerImageTag: "0.1.52"
-  documentationUrl: "https://docs.faros.ai"
+  dockerImageTag: "0.1.70"
+  documentationUrl: "https://github.com/faros-ai/airbyte-connectors/tree/main/sources/agileaccelerator-source"
```
Comment (tovbinm) [inline init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml:145]: I think it's ok to just have `https://github.com/faros-ai/airbyte-connectors/` as the docs for all these sources.

File: init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml [hunk 2]
```diff
@@ -181,15 +184,24 @@
         page_size:
           type: "integer"
           title: "Page Size"
+          description: "Page size to use when requesting records from AgileAccelerator\
```
Comment (tovbinm) [inline init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml:187]: remove multi line

File: init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml [hunk 3]
```diff
@@ -141,8 +141,8 @@
 - sourceDefinitionId: "05f9f7d4-d900-4ebd-89fc-abcce64e342b"
   name: "Agile Accelerator"
   dockerRepository: "farosai/airbyte-agileaccelerator-source"
-  dockerImageTag: "0.1.52"
-  documentationUrl: "https://docs.faros.ai"
+  dockerImageTag: "0.1.70"
```
Comment (tovbinm) [inline init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml:144]: we are at `0.1.72`

**Review by tovbinm — APPROVED** [review]

File: init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml (hunk 2 above)
Comment (thomas-gerber) [reply init/resources/airbyte/workspace/airbyte_config/STANDARD_SOURCE_DEFINITION.yaml:187]: This was generated by what is the container image when the source was manually imported into an Airbyte instance (and then exported to be part of the init config). Is it safe to change? @cjwooo ?

**Pull request merged**